### PR TITLE
Add recipe for Caelus Python Library

### PR DESCRIPTION
**Pull request by sayerhs**
Checklist
- [X] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [X] Source is from official source.
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [X] If static libraries are linked in, the license of the static library is packaged.
- [X] Build number is 0.
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [X] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by sayerhs** [comment]
@conda-forge-admin, please ping team

File: recipes/caelus/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  host:
+    - python >=3.6
+    - pip
+    - setuptools
+
+  run:
+    - python >=3.6
+    - six
+    - numpy
+    - matplotlib
```
Comment (xhochy) [inline recipes/caelus/meta.yaml:32]: `matplotlib-base` should be enough if you don't require explicitly the Qt backend (unlikely).
```suggestion
    - matplotlib-base
```

File: recipes/caelus/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    - pyyaml
+    - ply
+    - pytz
+    - jinja2
+
+test:
```
Comment (xhochy) [inline recipes/caelus/meta.yaml:38]: Add a check for requirements consistency.
```suggestion
test:
  requires:
    - pip
  commands:
    - pip check
```

File: recipes/caelus/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,54 @@
+{% set name = "caelus" %}
```
Comment (xhochy) [inline recipes/caelus/meta.yaml:1]: It is a bit confusing that the package on PyPI is a different one. Do you also distribute the package there (with a different name)?

Comment (sayerhs) [reply recipes/caelus/meta.yaml:1]: We are not distributing the package on PyPI. Do you want me to use a different package name?

Comment (xhochy) [reply recipes/caelus/meta.yaml:1]: I took a look at the PyPI package with the same name and it seems like a non-used and inactive project so I think in this special case it's fine to keep the name as-is.

**Pull request merged**